### DISCUSSION TITLE
Add ability to jump straight to a hit point in print statements

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#7133] [Ideas]
It's possible to be in a situation like this where 200 hits are tucked behind 20 pixels of content:
<img width="655" alt="image" src="https://user-images.githubusercontent.com/9154902/172984927-0b524ad1-feb8-440b-9567-dfbf426b7374.png">

Right now if you wanted to go to hit 200, you'd have to press the arrow key 199 times, which isn't ideal. We should discuss ways to make this easier.

My favourite design solution is to allow people to click directly into the hits capsule, like so ->

<img width="183" alt="image" src="https://user-images.githubusercontent.com/9154902/172986434-3d4bf29f-d2c1-4be9-a2b9-a9e7142c648d.png">

**Comment by jasonLaster** [comment]
First of all, i think it's important to be able to jump straight to a hit, so glad we're exploring.

* Not sure if a single click or double click makes sense. It should be discoverable with the right pointer / hover effect. What's some good precedent?
* It'd be nice to also have an option in a context menu or elsewhere. The easiest spot would be to add it to any context menu we would show for a single hit whether it is a marker or console message
